contrib/world_builder/CMakeLists.txt

```diff
@@ -16,35 +16,72 @@ if(POLICY CMP0058)
 endif()
 
 # load in version info and export it
-SET(WORLD_BUILDER_SOURCE_DIR ${CMAKE_SOURCE_DIR})
-INCLUDE("${CMAKE_SOURCE_DIR}/cmake/version.cmake")
+SET(WORLD_BUILDER_SOURCE_DIR ${PROJECT_SOURCE_DIR})
```

Comment:
Line 1 triggers a warning from my cmake (Compatibility with CMake < 2.8.12 will be removed from a future version of  CMake.). Requiring cmake 2.8.12 removes the warning and 2.8.12 was release in 2013, so I would guess there is no harm in increasing WorldBuilder's minimum cmake version to 2.8.12? ASPECT already requires cmake 3.1.0 so we could also just require a higher version for the worldbuilder version inside ASPECT.

Reply:
Currently the minimum cmake version in the world builder is 2.8.4, but that has been the case since probably version 0.3.0, so almost 3 year (but at least 2). I see no harm in, and I think it is a good idea to increase it to 2.8.12. I will make the changes.